app/utils/paginator.js

```diff
@@ -23,15 +23,16 @@ Paginator.MAX_PAGE_SIZE = LARGE_PAGE_SIZE
 
 /**
  * @param  {number} pageNumber
- * @param  {number} pageName
+ * @param  {string} pageName
```

Comment:
Good catch 👍  If we're going to keep this up to date I suppose we should include the boolean disabled param

Reply:
👍 Added `disabled` param